api/config/config_stable.json

```diff
@@ -21,6 +21,7 @@
   },
   "googleCloudStorageService": {
     "credentialsBucketName": "all-of-us-rw-stable-credentials",
+    "emailImagesBucketName": "all-of-us-workbench-test-email-images",
```

Comment:
Should these be specific to the environment?

Reply:
Well, the images don't vary, so didn't think so.